tutorials/inputs/inputevent.rst

```diff
@@ -194,10 +194,12 @@ There are several specialized types of InputEvent, described in the table below:
 |                                                                   | as feedback. (more on this below)       |
 +-------------------------------------------------------------------+-----------------------------------------+
 
-Actions
--------
+.. _doc_input_actions:
```

Comment:
Out of curiosity, what is this anchor needed for? I don't see any references to it in this PR (nor in the current docs in any file).

Reply:
I was planning to link to this section from other areas. But that's not done in this PR, so I can remove the anchor and only add it when it's actually used